<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Multi Screen</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: grey;</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>621</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>60</x>
           <y>20</y>
           <width>191</width>
           <height>141</height>
          </rect>
         </property>
         <property name="text">
          <string>Welcome in the Amazing game</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2">
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>40</y>
           <width>171</width>
           <height>61</height>
          </rect>
         </property>
         <property name="text">
          <string>Main menu</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>170</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Launch Game</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_4">
         <property name="geometry">
          <rect>
           <x>250</x>
           <y>170</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_3">
        <widget class="QLabel" name="label_3">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>50</y>
           <width>46</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </widget>
       <widget class="QWidget" name="page_4">
        <widget class="QLabel" name="player1">
         <property name="geometry">
          <rect>
           <x>110</x>
           <y>60</y>
           <width>46</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Player1</string>
         </property>
        </widget>
        <widget class="QLabel" name="player2">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>90</y>
           <width>46</width>
           <height>13</height>
          </rect>
         </property>
         <property name="text">
          <string>Player2</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>380</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Page 1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Page 2</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
